bin/Audio_Transcription.md

```diff
@@ -0,0 +1,38 @@
+## Transcribing longer audio clips
+
+### 0. Pre-requisites
+
+```
+pip3 install --user webrtcvad; pip3 install --user deepspeech; pip3 install --user tensorflow; pip3 install -r requirements.txt
```

Comment:
Likely better to document with a virtualenv rather than suggesting local install

Reply:
I had run into some import issues (module not found) when I had tried it with a virtualenv the first time I wrote this. Will give it another shot and give you an update about it, have a fairer understanding of python since then :P

Reply:
We use venv everywhere, so it's unlikely the error was from deepspeech itself

Reply:
Will look into it.